lib/observer/src/observer_wx.erl

```diff
@@ -776,12 +776,13 @@ get_nodes() ->
 
 %% see erl_epmd:(listen_)port_please/2
 erl_dist_port() ->
-    try
-        erl_epmd = net_kernel:epmd_module(),
-        {ok, [[StringPort]]} = init:get_argument(erl_epmd_port),
-        list_to_integer(StringPort)
-    catch
-        _:_ ->
+    case net_kernel:epmd_module() of
+        erl_epmd ->
+            case erl_epmd:listen_port_please(nonode, nohost) of
```

Comment:
Note the fake `nonode`, `nohost`, those arguments are ignored by `erl_epmd:listen_port_please/2`, and this way we use the public API. Alternatively we could expose `erl_epmd:erl_epmd_listen_port/0` and make it `-doc false`.

Reply:
We could also expose it as `net_kernel:epmd_listen_port`, to mirror `net_kernel:epmd_module`.

Reply:
`epmd_module` is a broader config, while `erl_epmd_listen_port` is specific to `erl_epmd`, as per the other comment.